src/main/java/com/simanavets/tenderflexbackend/service/UserService.java

```diff
@@ -0,0 +1,5 @@
+package com.simanavets.tenderflexbackend.service;
+
+public interface UserService {
```

Comment:
Why you need interface without available methods? If you implement some interface you must override it's methods bit in this case there are nothing to override. Please add all methods relevant to UserServise interface, and then add @Override annotation to all overridden methods in the implementation